lnbits/core/crud.py

```diff
@@ -77,7 +77,7 @@ async def get_user(user_id: str, conn: Optional[Connection] = None) -> Optional[
                 SELECT balance FROM balances WHERE wallet = wallets.id
             ), 0) AS balance_msat
             FROM wallets
-            WHERE "user" = ?
```

Comment:
- why was this done in this PR, it supposed to be a refactoring only
- this should be done in a different PR

Reply:
else deleted wallets will show up on the wallet page. i directly uses the get_user crud to render it into jinja html template.